File: Hyeseung/Programmers/조건에 맞게 수열 변환하기1.java
package Hyeseung.Programmers;

class Solution {
    public int[] solution(int[] arr) {
        int count = 0;
        for (int a : arr) {
            if (a % 2 == 0 && a >= 50)
                arr[count] = a / 2;
            else if (a % 2 != 0 && a < 50)
                arr[count] = a * 2;
            count++;
        }
        return arr;
    }
}

File: 2st/Riyeon/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    for i in range(len(arr)):
        if arr[i] >= 50 and arr[i] % 2 == 0:
            arr[i] /= 2
        elif arr[i] < 50 and arr[i] % 2 == 1:
            arr[i] *= 2
    return arr